nmrfx-analyst-gui/src/main/java/org/nmrfx/analyst/gui/tools/SimplePeakRegionTool.java

```diff
@@ -231,10 +232,18 @@ public void findRegions() {
                 // normalize to the smallest integral, but don't count very small integrals (less than 0.001 of max
                 // to avoid artifacts
 
-                double threshold = regions.stream().mapToDouble(r -> r.getIntegral()).max().orElse(1.0) / 1000.0;
-                regions.stream().mapToDouble(r -> r.getIntegral()).filter(r -> r > threshold).min().ifPresent(min -> {
-                    dataset.setNorm(min * dataset.getScale() / 1.0);
-                });
+                double threshold = regions.stream().mapToDouble(DatasetRegion::getIntegral).max().orElse(1.0) / 1000.0;
+                OptionalDouble min = regions.stream().mapToDouble(DatasetRegion::getIntegral).filter(r -> r > threshold).min();
+                if (min.isEmpty()) {
+                    // if all the integrals are negative, get the smallest absolute value
+                    min = regions.stream().mapToDouble(r -> Math.abs(r.getIntegral())).filter(r -> r > threshold).min();
+                }
+                if (min.isPresent()) {
+                    // Save absolute value as the norm, so that spectra with all negative integrals will appear consistent
+                    // with mixed and all positive integral spectra
+                    dataset.setNorm(min.getAsDouble() * dataset.getScale());
+                }
+
```

Comment:
Used the smallest *Positive* peak for the norm value for all positive or mixed positive and negative peaks. Is this okay? Or should I change to find the smallest peak from both the positive and negative peaks (absolute value)?